GOVERNANCE.md

```diff
@@ -13,7 +13,7 @@
 ## Membership Policy
 
 This document is primarily concerned with governance for project maintainers, as well as Special Interest Groups (SIGs), Subprojects, and Working Groups (WGs).
-We have a separate [membership policy](.membership-policy.md) document that details our contributor ladder and criteria for our other roles, such as new members, reviewers, approvers, SIG and WG chairs, and Subproject leads.
```

Comment:
Why loose this link?

Reply:
The link reference is at the bottom of file as it's used in a couple of places. The replacing line uses it instead.

Reply:
+1